### PR TITLE
Functional tests - Refacto customers tests

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Refacto customers tests
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | TEST_PATH="functional/BO/04_customers/01_customers/*" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25189)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/04_customers/01_customers/01_filterAndQuickEditCustomers.js [hunk 1]
```diff
@@ -199,11 +208,33 @@ describe('Filter And Quick Edit Customers', async () => {
         await expect(numberOfCustomersAfterReset).to.equal(numberOfCustomers);
       });
     });
+
+    it('should filter by registration \'Date from\' and \'Date to\'', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterByDate', baseContext);
+
+      // Filter orders
```
Comment (boubkerbribri) [inline tests/UI/campaigns/functional/BO/04_customers/01_customers/01_filterAndQuickEditCustomers.js:215]: Just a question, why we need this ?

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/04_customers/01_customers/01_filterAndQuickEditCustomers.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/04_customers/01_customers/01_filterAndQuickEditCustomers.js:215]: To test filter by registration date

**Review by boubkerbribri — DISMISSED** [review]

**Review by NeOMakinG — DISMISSED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
## WAIT FOR 1780 RELEASE

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**